docs/dev/zksync-v2/contracts.md

```diff
@@ -4,14 +4,14 @@ To maintain the same security as L1, the zkSync operator must publish the contra
 
 While deploying contracts for the first time may be relatively expensive, factories, which deploy contracts with the same code multiple times, can have huge savings compared to L1.
 
-All these specifics make the process of deploying smart contracts on zkEVM comply with the major rule: _The operator should know the code of the contract before it is deployed_. That means that deploying contracts is only possible by the means of `EIP712` transactions with the `factory_deps` field containing the supplied bytecode. More on EIP712 transactions [here](../../api/api.md#eip712).
+All these specifics make the process of deploying smart contracts on zkEVM comply with the major rule: _The operator should know the code of the contract before it is deployed_. That means that deploying contract is only possible by the means of `EIP712` transactions with the `factory_deps` field containing the supplied bytecode. More on EIP712 transactions [here](../../api/api.md#eip712).
```

Comment:
`deploying contract` sounds weird to me. I guess that `deploying contracts` is better here.

Reply:
get you, removed this change.

Reply:
You should update it here as well (under the comment). 

After that is done, I can merge it